### ISSUE TITLE
belang van een top level ontologie

### ISSUE THREAD

**Issue by PalmJanssen** [Geonovum/NEN3610-Linkeddata#73]
Door Mick Baggen
## Waar

Par.   5.5 (revisited)

## Samenvatting

Veel   van de beschreven modelleerverschillen tussen UML en OWL worden nu aan de   beide talen zelf toegeschreven, maar zijn in veel gevallen toe te schrijven   aan het ontbreken van een fatsoenlijke Top Level (Foundational, Upper)   Ontology.

## Beschrijving

In   zo'n ontologie kunnen veel misverstanden, zoals het vermengen van fysieke   objecten en de tekens, waarmee ze worden aangeduid, worden vermeden. Een   ander veel voorkomend probleem, de ontologische vermenging van een fysiek   object (zeg: een persoon) en de rol van een object (zeg: vader, kind,   manager) kan daarmee ook worden aangepakt.

## Oplossingsrichting

Voer   een Top Level Ontology in met een veel hoger abstractie niveau dan dat van de   huidige NEN 3610 (en de onderliggende IMxxxx modellen), en gebruik dat als   basis voor je conceptueel modellen (zie eerdere opmerking).

**Comment by pmaria** [comment]
Ja, dit kun je doen. En in principe hebben we dit ook gedaan bij het formuleren van de NEN 3610 ontologie in OWL en het relateren aan andere standaard ontologieen zoals GeoSPARQL, PROV, FOAF, etc.. Hoewel, nog vrij impliciet.

Echter, ik zie niet in hoe dit helpt bij het probleem van de modelleerverschillen tussen bestaande UML modellen en goede LD doelmodellen.

Het meerendeel van de bestaande UML modellen blijven een met impliciete kennis gestructureerde technische weergave van het UOD. Het hebben van een topontologie veranderd daar niks aan.

**Comment by architolk** [comment]
Als ik aan upper ontology denk, dan denk ik al snel aan iets dat begint met "Thing" en dan afdaalt. Maar feitelijk doet NEN3610 dit in een gedeelte van het model (dat als een geoobject is, en vervolgens afdaalt). Aangezien dit er al in zit, wordt vermoedelijk iets anders bedoeld, maar ik begrijp dan even niet wat...

**Comment by PalmJanssen** [comment]
Ik lees de opmerking als volgt.
Een model op hoger abstractie niveau kan er bijvoorbeeld als volgt uitzien:
Het hoogste niveau is een ding. Dat ding kan rollen aannemen die het vervult in relatie tot ander dingen. Een mens is een ouder in relatie tot een kind, een leerling in relatie tot een meester enz. Een ding kan getypeerd worden door zijn materiële kenmerken maar een ding kan ook getypeerd worden door de functie die het vervult. Als een dergelijk topmodel stevig in elkaar zit dan helpt dat bij het maken van correcte domeinmodellen.
Bij NEN 3610 is dit topmodel zwak ontwikkeld en is er daarom veel wijsheid nodig bij domeinmodellen.

**Comment by lvdbrink** [comment]
De top level van CB-NL is zoiets. Kan me herinneren dat we daar uitgebreid mee bezig zijn geweest. AL zie ik Rol daar niet terug... 

Ander voorbeeld is [DOLCE ultralight](http://www.ontologydesignpatterns.org/ont/dul/DUL.owl). 

Misschien iets om mee te nemen in de nieuwe versie van NEN3610.

**Comment by NielsHoffmann** [comment]
De nta8035 en vooral de nen2660 beoogt een toplevel ontologie voor de gebouwde omgeving te realiseren...
________________________________

**Comment by lvdbrink** [comment]
Zie ook het [issue hierover](https://github.com/Geonovum/bmgi/issues/29) in de NEN3610 Github (BMGI).

**Comment by JoepvanGenuchten** [comment]
We hebben dit onderwerp bij de review sessie besproken en zijn tot ongeveer de volgende conclusie gekomen. 

De vraag om een upper ontology is zeer voor de hand liggend, echter op het moment van schrijven waren hier beperkte mogelijkheden toe.
In de periode dat  NEN3610-ld geschreven werd, werd tegelijkertijd ook aan het MIM gewerkt. Deze 2 projecten hebben elkaar inhoudelijk sterk beïnvloed. Zo heeft hebben de inzichten van NEN3610-ld er voor gezorgd dat het huidige MIM minder aan Object Orientatie gebonden is. Op het moment van schrijven was MIM nog niet gespecificeerd. Tijdens het opstellen van dit document is er dus voor gekozen om de NEN3610 specificatie (de text) als, in natuurlijke taal beschreven, referentie te gebruiken. Met de kennis van nu zouden we zeer waarschijnlijk stellen dat een NEN3610 upper ontology conform het MIM gemodelleerd zou worden, van waaruit de mapping naar linked data gemaakt zou kunnen worden.
